src/DonationForms/V2/resources/components/Migration/index.tsx

```diff
@@ -0,0 +1,48 @@
+import {__, sprintf} from "@wordpress/i18n";
+import {createInterpolateElement} from "@wordpress/element";
+import {CheckVerified} from "@givewp/components/AdminUI/Icons";
+
+/**
+ * @unreleased
+ */
+export const UpgradeModalContent = () => {
+
+    const {supportedAddons, supportedGateways} = window.GiveDonationForms;
+
+    return <p style={{maxWidth: '400px'}}>
```

Comment:
Does this need to be wrapped in parenthesis?

Reply:
The use of parenthesis in the return statement is to inform the compiler that the value of the return starts on the next line. It isn't strictly necessary, except for formatting.